Updated README.md

a-velasco opened the pull request:
## Issue
README.md is out of sync with latest updates to the [MongoDB home page on charmhub.io](https://charmhub.io/mongodb)
Jira issue: [DPE-3747](https://warthogs.atlassian.net/browse/DPE-3747)

## Solution
- Edited the text to match home page more closely
- Removed unnecessary or repeated information
- Added further links and resources 

[DPE-3747]: https://warthogs.atlassian.net/browse/DPE-3747?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

a-velasco committed:
Updated README.md

MiaAltieri reviewed (dismissed):
this looks excellent and your work is greatly appreciated!

 Be sure Michelle reviews this - she was the one who wrote this initially.

Also I believe the text here should match the overview page on the charmhub docs

michtabirao committed:
Update README.md (#385)

updated the readme

## Issue
legal info 
List of features not complete

## Solution
legal info updated
List of features updated

a-velasco committed:
nit

a-velasco committed:
Merge branch '6/edge' into readme-update-ga

MiaAltieri commented on README.md:
```diff
@@ -5,59 +5,46 @@
-
-Aside from a VM operator, Canonical also developed another operator called [Charmed MongoDB (K8s operator)](https://charmhub.io/mongodb-k8s?channel=5/edge) that operates in the Kubernetes environment, and you can see the documentation [here](https://charmhub.io/mongodb-k8s?channel=5/edge).
+It currently supports:
+* MongoDB deployment
```
I would remove and/or reword

MiaAltieri commented on README.md:
```diff
@@ -5,59 +5,46 @@
+* MongoDB deployment
+* Replication
+* Sharding
+* Password set and rotation
```
```suggestion
* Password rotation
```

MiaAltieri commented on README.md:
```diff
@@ -5,59 +5,46 @@
+* Sharding
+* Password set and rotation
+* User management
+* Client connections via integrations
```
```suggestion
* Client connections
```

MiaAltieri commented on README.md:
```diff
@@ -5,59 +5,46 @@
+* TLS encryption
+* Backup & restore
+* Monitoring
+* Auditing
```
```suggestion
* Audit logging
```

MiaAltieri commented on README.md:
```diff
@@ -5,59 +5,46 @@
+* Backup & restore
+* Monitoring
+* Auditing
+* Observability
```
```suggestion
```
remove this - same thing as monitoring

a-velasco committed:
Review updates
